Packs/cyberark_AIM/Integrations/CyberArkAIM_v2/CyberArkAIM_v2.py

```diff
@@ -152,7 +157,7 @@ def main():
             'fetch-credentials': fetch_credentials
         }
         if command in commands:
-            return_results(commands[command](client))  # type: ignore[operator]
+            return_results(commands[command](client, demisto.args()))  # type: ignore[operator]
```

Comment:
`list_credentials_command` and `test_module ` doesn't expect the args.

Reply:
I don't know why I thought it shouldn't be a problem but it obviously is.